src/reconcile/rec_visibility.c

```diff
@@ -133,7 +137,8 @@ __rec_append_orig_value(
     __wt_scr_free(session, &tmp);
     /* Free append when tombstone allocation fails */
     if (ret != 0) {
-        __wt_free_update_list(session, &append);
+        __wt_free(session, tombstone);
```

Comment:
I prefer the order to be in reverse with respect to allocation, so could you swap these two statements.

Reply:
Fixed